test/events.test.js

```diff
@@ -0,0 +1,251 @@
+import chai from 'chai';
+import request from 'supertest';
+import bcrypt from 'bcrypt';
+import Server from '../server';
+import mongo from '../server/common/mongo';
+import userModel from '../server/models/user';
+import eventModel from '../server/models/event';
+
+
+const expect = chai.expect;
+
+chai.use(require('chai-like'));
+chai.use(require('chai-things'))
+
+describe('Event Actions', function() {
+  let token;
+
+  before(async () => {
+
+    //creating a user
+    const userPassword = 'aasf_iiitm';
+    const saltRounds = 10;
+    const password = await bcrypt.hash(userPassword, saltRounds);
+    await mongo();
+    await userModel.create({
+      _id: '2018BCS-000',
+      name: 'AASF',
+      role: 'admin',
+      password,
+    });
+    
+    //logging in the user
+    const result = await request(Server)
+    .post('/users/login')
+    .send({ roll: '2018BCS-000', password: 'aasf_iiitm' });
+    token = result.body.token;
+  })
+
+  //Create an event object before each test
+  beforeEach(async () => {
+    
+    //creatiing an event
+    await eventModel.create({
+      name: "webkriti",
+      startDate: "07-10-2021",
+      numberOfDays: 2,
+      category: "technical",
+      importance: 0
+    })
+    
+  })
+
+  afterEach(async () => {
+    await eventModel.deleteMany({});
+  })
+  
+  after(async () => {
+    await userModel.deleteMany({});
+  });
+
+  //Test for fetching events
+  it('should get events', async() => {
+  
+    let response = await request(Server)
+    .get('/admin/events')
+    .set('Authorization', `bearer ${token}`);
+
+    expect(response.body).to.be.an('object');
+    expect(response.status).to.equal(200);
+    expect(response.body.message).to.be.an('string');
+    expect(response.body.events).to.be.an('array')
+    expect(response.body.events[0].name).to.be.a('string')
+    expect(response.body.events[0].startDate).to.be.a('string')
+    expect(response.body.events[0].numberOfDays).to.be.a('string')
```

Comment:
I believe this is throwing an error because `numberOfDays` is actually a number. I think you expected it to be a string because of the documentation. Can you please change it to `number` datatype in the documentation too?

Let me know if you can't find the place where to make the change.

Reply:
Sir, I have set the data type as number in the test. It would be great if you can suggest where I should make changes to fix the documentation.